osi_sensordata.proto

```diff
@@ -73,6 +73,22 @@ message DetectedEntityHeader
         // Sensor temporary available.
         //
         DATA_QUALIFIER_TEMPORARY_AVAILABLE = 6;
+
+        // Reported data are from the sensor in a restricted view and performance.
+        //
+        DATA_QUALIFIER_REDUCE_INVIEW_AND_PERFORMANCE = 7;
```

Comment:
REDUCED_VIEW???

Reply:
has COVERAGE from ISO been renamed to VIEW?

Reply:
ISO and ADI both uses REDUCE

Yes, the name has changed, I will ask why.

Reply:
ViEW is also a mistake. I will change it to COVERAGE.